pwndbg/gdblib/events.py

```diff
@@ -163,6 +228,10 @@ def caller(*a: P.args, **kw: P.kwargs) -> None:
     registered[event_handler].setdefault(priority, []).append(caller)
     if event_handler not in connected:
         handle = partial(invoke_event, event_handler)
+
+        if event_handler == gdb.events.new_objfile:
```

Comment:
I bet this can be `event_handler is gdb.events.new_objfile` but idk, maybe not?

Reply:
don't know :)